react/details/public/index.html

```diff
@@ -14,10 +14,16 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Conference details</title>
+    <script>
```

Comment:
is this simulating what the login widget should do?

Reply:
the server/backend should append the Keyclock server details for the client